docs/packaging.md

```diff
@@ -0,0 +1,71 @@
+# Packaging the Carbon Aware SDK
+
+With the addition of the C# Client Library as a way to consume the Carbon Aware SDK, we have also added [bash scripts](../scripts/package/) to package the library, and have included a sample [Console App](../samples/lib-integration/) showing how the package can be consumed.
+
+## Included Projects
+
+The current package include 8 projects from the SDK:
+
+1. "GSF.CarbonAware"
+2. "CarbonAware"
+3. "CarbonAware.Aggregators"
+4. "CarbonAware.DataSources.Json"
+5. "CarbonAware.DataSources.Registration"
+6. "CarbonAware.DataSources.WattTime"
+7. "CarbonAware.LocationSources.Azure"
+8. "CarbonAware.Tools.WattTimeClient"
+
+These 8 projects enable users of the library to consume the current endpoints exposed by the library. The package that needs to be added to a new C# project is `GSF.CarbonAware`.
+
+## Included Scripts
+
+There are 2 scripts included to help the packaging process
+
+1. `create_package.sh <dotnet_solution> <package_destination>`
+2. `add_package.sh <dotnet_project> <package_destination>`
+
+The [`create_package`](../scripts/package/create_packages.sh) script is called with 2 parameters: the CarbonAwareSDK dotnet solution file (`.sln`) path, and the output directory destination for the package. The [`add_package`](../scripts/package/add_package.sh) script is also called with 2 parameters: the target project file (`.csproj`) path, and the package destination path.
+
+To see a working example of both scripts being invoked, you can look at the github action detailed in [build-packages.yaml](../.github/workflows/build-packages.yaml).
+
+### Running the packaging scripts
```

Comment:
Consider putting this section first, since it reflects the requirements, and then moving the above instructions into their own section after it

EG 
```markdown
## Included Scripts

### Requirements
<the docker/vscode/remotecontainers stuff>

### Running the Scripts

There are 2 scripts included to help the packaging process... <etc>
```